### PR TITLE
Adding Previous Salary exchaned amount

### LINKED ISSUE #68: Exchange rate

### PR DESCRIPTION
**Pull request by josx**
closes #68

### CODE REVIEW

File: future-apps/payroll/app/src/screens/components/PreviousSalary.js [hunk 1]
```diff
@@ -107,8 +107,25 @@ const StyledFilters = styled.div`
   }
 `
 
+function getExchangedAmount (payment) {
+  return (payment.amount.amount / payment.exchangeRate.amount)
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/screens/components/PreviousSalary.js:111]: Wouldn't be more clear if we had just `payment.exchangeRate`?

File: future-apps/payroll/app/src/screens/components/PreviousSalary.js [hunk 2]
```diff
@@ -107,8 +107,25 @@ const StyledFilters = styled.div`
...
 
-function mapStateToProps ({ salaryAllocation = [], employees = [], accountAddress = {}, payments = [], denominationToken = [] }) {
+function mapStateToProps ({
+  salaryAllocation = [],
+  employees = [],
+  accountAddress = {},
+  payments = [],
+  denominationToken = []
+}) {
+  payments = parsePayments(payments)
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/screens/components/PreviousSalary.js:128]: Nice!

File: future-apps/payroll/app/src/screens/components/PreviousSalary.js (hunk 1 above)
Comment (josx) [reply future-apps/payroll/app/src/screens/components/PreviousSalary.js:111]: Yes, could be but We dont know if they want to show or if we need to use exchangeRate (as is right now) for another calculation or to show it

File: future-apps/payroll/app/src/screens/components/PreviousSalary.js (hunk 2 above)
Comment (josx) [reply future-apps/payroll/app/src/screens/components/PreviousSalary.js:128]: Anyways in #81 I have refactored  this and I put it into store/marshalling.js

**Review by lmcorbalan — APPROVED** [review]
👍

**Pull request merged**